added house - Android tool (#1558)
OWASP/owasp-mstg: Document/0x05b-Basic-Security_Testing.md

In the end, it is up to you to decide where would you like to work with the data. Sometimes it will be more convenient to do it from JavaScript and in other cases Python will be the best choice. Of course you can also send messages from Python to JavaScript by using `script.post`. Refer to the Frida docs for more information about [sending](https://www.frida.re/docs/messages/#sending-messages-from-a-target-process "Sending messages from a target process") and [receiving](https://www.frida.re/docs/messages/#receiving-messages-in-a-target-process "Receiving messages in a target process") messages.

##### House

> [House](https://github.com/nccgroup/house "House") is a runtime mobile application analysis toolkit for Android Apps, developed and maintained by NCC Group and is written in Python.

Reviewer: ```suggestion
[House](https://github.com/nccgroup/house "House") is a runtime mobile application analysis toolkit for Android apps, developed and maintained by the NCC Group and is written in Python.
```

> It's leveraging on a running Frida Server on a rooted device or the Frida Gadget in a repackaged Android App. The intention of House is to allow an easy way for prototyping of Frida scripts by offering a Web GUI.

Reviewer: Consistency: server/gadget/app with the rest of the guide + little fixes
```suggestion
It's leveraging a running Frida server on a rooted device or the Frida gadget in a repackaged Android app. The intention of House is to allow an easy way of prototyping Frida scripts via its convenient web GUI.
```
